source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -10,6 +10,7 @@
         if (data) {
             this.fromJs(data);
         }
+        this.saveButtonEnable = ko.observable('');
```

Comment:
It looks as a boolean flag, right? in that case what do you think about to initialize it as `false` in place of empty string?